doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -156,7 +156,7 @@ In order for the networking solution(s) to be conformant with the Reference Arch
 
 |Ref|Specification|Details|Requirement Trace|
 |---|---|---|---|
-|`ra2.ntw.001`|CNI multiplexer/metaplugin|As the selected CNI multiplexer/metapulgin MUST support other CNI plugins (`req.inf.ntw.06`) and SHOULD provide an API based solution to administer the networks from a central point (`req.inf.ntw.03`) the selected CNI multiplexer/metapulgin may be [DANM](https://github.com/nokia/danm). For more detailed feature comparision of CNI multiplexers/metaplugins see Table 4-5| [req.inf.ntw.06](./chapter02.md#23-kubernetes-architecture-requirements), [req.inf.ntw.03](./chapter02.md#23-kubernetes-architecture-requirements) |
+|`ra2.ntw.001`|CNI multiplexer/metaplugin|The selected CNI multiplexer/metapulgin MUST be conformant with the CNI spec, MUST support other CNI plugins (req.inf.ntw.06), and SHOULD provide an API based solution to administer the networks from a central point (req.inf.ntw.03). Example CNI implementations which meet these requirements include:<br> - DANM<br> - Multus.<br><br>For more detailed feature comparision of CNI multiplexers/metaplugins see Table 4-5| [req.inf.ntw.06](./chapter02.md#23-kubernetes-architecture-requirements), [req.inf.ntw.03](./chapter02.md#23-kubernetes-architecture-requirements) |
```

Comment:
Let's wait until #1874 concluded.

Reply:
@CsatariGergely are you ok with this, given:
- the discussion we had at TSC
- I will raise an issue I will raise to document the gap in abstracted management of multi-NIC solutions
- we remove the CRD requirement from #1872

Reply:
Yes if we somehow enable deferring from [principle 8](https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#overall-principles) and we list the gaps of Multus compared to the CNTT requirements (no central IPAM, non abstract API (DANM has an abstract API))

Reply:
@CsatariGergely  I am sure the IPAM requirements can be satisfied by a stand-alone CNI IPAM plugin. W.r.t. the meta plugin again it may not be necessary to create another set of API but just conform to the CNI APIs.

Reply:
IPAM issue: The problem with Muliplexers is that they manage multiple CNI-s, therefore without a central IPAM they are not capable to guarantee that there are no overlaps in the IP allocations. At the moment there is no mention about a separate central IPAM in the component list, so with Multus RA2 is incomplete.
API issue: CNI API is not something what you can manage centrally while req.inf.ntw.03 requires central administration and configuration. 

We should list all of these defects of the Multus based solution.

Reply:
@CsatariGergely The point is that there are requirements for IPAM, Security Policies, Multiplexers, etc. and these are all separate components needed for an overall networking solution. They are not defects in Multus or for that matter any other solution.